site/_data/i18n/common.yml

```
@@ -78,12 +78,9 @@ success:
   en: 'Success'
   es: 'Éxito'
 
-gotchas:
-  en: 'Gotchas'
-  es: 'Gotchas'
-
 important:
   en: 'Important'
+  es: 'Importante'
```

Single words are safe to be translated by Google Translate.

```suggestion
es: Importante
ja: 重要
ko: 중요한
pt: Importante
ru: Важный
zh: 重要的
```